yarn-project/prover-node/src/tx-provider/aztec-node-tx-provider.ts

```diff
@@ -7,4 +7,8 @@ export class AztecNodeTxProvider implements TxProvider {
   getTxByHash(txHash: TxHash): Promise<Tx | undefined> {
     return this.node.getTxByHash(txHash);
   }
+
+  addEpochProofQuote(quote: EpochProofQuote): Promise<void> {
```

Comment:
This functionality doesnt feel like it fit with the class name

Reply:
Yeah agreed. I'll rework it.

Reply:
I just changed the name of the interface to `ProverCoordination`. I think it actually fits well, since right now the coordination takes place via a wrapped aztec node, but that will be replaced by an implementation that uses p2p as the mechanism for coordination.